internal/datacoord/compaction_task_clustering.go

```diff
@@ -293,6 +285,10 @@ func (t *clusteringCompactionTask) processExecuting() error {
 }
 
 func (t *clusteringCompactionTask) processMetaSaved() error {
+	if len(t.GetTmpSegments()) == 0 {
+		log.Info("tmp segments is nil, skip stats task", zap.Int64("planID", t.GetPlanID()))
```

Comment:
Add comments to clarify this special compatibility handling.

Reply:
ok